### PR TITLE
RavenDB-15446 Studio conventions

### PR DESCRIPTION
**Pull request by hiddenshadow21**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-15446

### Additional description

New workflow checking RavenDB Studio conventions.

### Type of change

- New feature

### How risky is the change?

- Not relevant

### Backward compatibility

- Not relevant

### Is it platform specific issue?

- No

### Documentation update

- No documentation update is needed 

### Testing by Contributor

- It has been verified by manual testing

### Testing by RavenDB QA team

- No special testing by RavenDB QA team is needed

### Is there any existing behavior change of other features due to this change?

- No

### UI work

- No UI work is needed

### CODE REVIEW

File: .github/workflows/studioConventions.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - name: Prettier Studio
+      if: always()
+      run: |
+        if [[ -n $GITHUB_HEAD_REF ]]; then
+          branch=$GITHUB_HEAD_REF
+        else
+          branch=$GITHUB_REF_NAME
+        fi
+
+        [[ ${branch:1:1} -lt 6 ]] && echo "skipping priettier for versions below 6.0" && exit 0
```
Comment (ml054) [inline .github/workflows/studioConventions.yml]: prettier

Comment (gregolsky) [reply .github/workflows/studioConventions.yml]: what do you mean?

Comment (gregolsky) [inline .github/workflows/studioConventions.yml]:
```suggestion
        [[ ${branch:1:1} -lt 6 ]] && echo "skipping prettier for versions below 6.0" && exit 0
```

**Review by gregolsky — APPROVED** [review]

**Comment by gregolsky** [comment]
@ml054 It seems to fail at the moment. Is this expected? https://github.com/ravendb/ravendb/actions/runs/4754892613/jobs/8448293983?pr=16314

**Comment by ml054** [comment]
Yes, 

Due to lack of eslinter we some errors - I fix those after merge.

**Pull request merged**